.github/contribute.md

```diff
@@ -36,6 +36,14 @@ It’s probably a good idea to first post a question or open an issue to report
 bug or suggest a new feature before creating a pull request.
 See [Project][] for more info.
 
+## `napi` folder
+
+This is the package that handles NAPI bindings for markdown-rs. It also supports WASM. It uses napi-rs to generate the npm packages published on npm.
+
+Tests use vitest, expect().toMatchInlineSnapshot() mostly. Most tests should be run with `pnpm vitest --run -u` to update snapshots, then read the test file again and make sure they have the expected result.
```

Comment:
I'd recommend `npm` first over requiring one of the many alternative package managers (yarn, pnpm, jsr, etc).

Reply:
most popular projects use pnpm now, it's become the standard for workspaces and a must as soon as you hit one of the many bugs in npm, which are many when using workspaces

Reply:
> most popular projects use pnpm now

That is objectively false.
None of the top 10 packages by download use pnpm.
And most in the top 1000 don't either.
https://github.com/wooorm/npm-high-impact

More important, unified, which this is broadly part of does not https://github.com/unifiedjs/unified

> soon as you hit one of the many bugs in npm

I hit far more pnpm bugs than npm

> barely maintained now

Last release less than a week ago https://www.npmjs.com/package/npm?activeTab=versions
And they come at a regular cadence.